tests/subsys/net/lib/nrf_cloud/fota_common/src/fmfu.c

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include "main.c"
+
+/* Verify that nrf_cloud_fota_fmfu_dev_set returns failure when the function parameter is NULL. */
+ZTEST(nrf_cloud_fota_common_test, test_00_nrf_cloud_fota_fmfu_dev_set_null_fdev)
+{
+	Z_TEST_SKIP_IFDEF(CONFIG_DFU_TARGET_FULL_MODEM_USE_EXT_PARTITION);
```

Comment:
Why is this skipped if CONFIG_DFU_TARGET_FULL_MODEM_USE_EXT_PARTITION=y?

Reply:
https://github.com/nrfconnect/sdk-nrf/blob/2006366ca2bc8437bbebbcb7072413abe064dd3f/subsys/net/lib/nrf_cloud/src/nrf_cloud_fota_common.c#L54

because it is testing the code in that if statement

Reply:
Got it.